docs/conf.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+
+from discord.ext.ipc import __version__ as version
+
+
+# -- Project information -----------------------------------------------------
```

Comment:
These comments can be removed too.

Reply:
Isn't this likely to fail cause the package isn't installed at this point?

Reply:
The sys path is pointed at the directory above, so it should just read the code alongside it. Since 'the repo' is cloned that path exists locally